array_api_strict/_helpers.py

```diff
@@ -0,0 +1,28 @@
+"""Private helper routines.
+"""
+import numpy as np
+
+from ._flags import get_array_api_strict_flags
+
+
+_py_scalars = (bool, int, float, complex)
+
+
+def _maybe_normalize_py_scalars(x1, x2):
+    from ._array_object import Array
+
+    flags = get_array_api_strict_flags()
+    if flags["api_version"] < "2024.12": # XXX: string comparison for versions
```

Comment:
I've been using string comparisons everywhere for versions. It should be fine since every array API version is `202X.12`.